closed/make/common/Modules.gmk

```diff
@@ -29,6 +29,9 @@ MODULES_FILTER += \
     jdk.internal.vm.ci \
     jdk.internal.vm.compiler \
     jdk.internal.vm.compiler.management \
+    jdk.jfr \
```

Comment:
The copyright date needs to be updated in this file.

Reply:
Updated.  Thanks for spotting.